Gems/AtomLyIntegration/EditorModeFeedback/Code/Source/Pass/State/FocusedEntityState.cpp

```diff
@@ -49,8 +49,11 @@ namespace AZ::Render
     }
     void FocusedEntityState::OnEditorModeDeactivated(
         [[maybe_unused]] const AzToolsFramework::ViewportEditorModesInterface& editorModeState, AzToolsFramework::ViewportEditorMode mode)
-    {   
-        m_inFocusMode = !(mode == AzToolsFramework::ViewportEditorMode::Focus);
+    {
+        if (m_inFocusMode)
+        {
+            m_inFocusMode = !(mode == AzToolsFramework::ViewportEditorMode::Focus);
```

Comment:
question: Could this be simplified to

```
if (m_inFocusMode)
{
    m_inFocusMode = mode != AzToolsFramework::ViewportEditorMode::Focus;
}
```
?

Reply:
+1, but I'd still use parenthesis for readability:

```
if (m_inFocusMode)
{
    m_inFocusMode = (mode != AzToolsFramework::ViewportEditorMode::Focus);
}
```

Reply:
I personally think parens are overused 😉 (in Rust you don't even need them around if statements and I think the code looks clearer 😛) For precedence things they of course make sense but when they're redundant I like to avoid them, but I'm happy to disagree and commit on this one 🙂

Reply:
fixed :+1